.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -3,9 +3,14 @@ name: Pull request
 about: Create a pull request
 label: 'triage me'
 ---
-Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+Thank you for opening a Pull Request!
+Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
 - [ ] Make sure to open a GitHub issue as a bug/feature request before writing your code! That way we can discuss the change, evaluate designs, and agree on the general idea
 - [ ] Ensure the tests and linter pass
```

Comment:
Could we add the Gradle command here?

```suggestion
- [ ] Ensure the tests and linter pass (`./gradlew spotlessApply` to automatically apply formatting)
```

Reply:
Auto for before it which tab for touch to get first Word.